python/cuml/dask/linear_model/ridge.py

```diff
@@ -0,0 +1,194 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+from cuml.dask.common import extract_ddf_partitions, extract_colocated_ddf_partitions, workers_to_parts
+from cuml.dask.common import to_dask_cudf
+from cuml.dask.common import raise_exception_from_futures
+from cuml.dask.common.comms import worker_state, CommsContext
+
+from collections import OrderedDict
+
+from dask.distributed import default_client
+from dask.distributed import wait
+
+from functools import reduce
+
+from uuid import uuid1
+
+
+class Ridge(object):
```

Comment:
This file is almost identical to the linear_regression one above. Can we just have one MNMGLinearModel class that handles both and does something like

`self.model = cumlRidge if model_type=="ridge" else cumlLinearRegression`

?

Reply:
I'll leave solving the code duplication problem to version 0.13 as we agreed.